server/src/core/server/services/dsaReports/reports.ts

```diff
@@ -254,6 +254,7 @@ export async function makeDSAReportDecision(
         targetUserID: rejectedComment.authorID,
         type: GQLNOTIFICATION_TYPE.ILLEGAL_REJECTED,
         comment: rejectedComment,
+        previousStatus: comment.status,
```

Comment:
these are just myself doing clean up to add in the previousStatus where possible so we can trust it if it's made available to us inside the context for future notifications work.